packages/blitz/src/bin/cli.ts

```diff
@@ -2,72 +2,102 @@ import * as path from "path"
 import resolveFrom from "resolve-from"
 import pkgDir from "pkg-dir"
 import chalk from "chalk"
-import { parseSemver } from '../utils/parse-semver'
+import { parseSemver } from "../utils/parse-semver"
+import envinfo from 'envinfo'
+import hasYarn from "has-yarn"
 
-console.log(
-  chalk.yellow(
-    `You are using alpha software - if you have any problems, please open an issue here:
-  https://github.com/blitz-js/blitz/issues/new/choose\n`,
-  ),
-)
 
-if (parseSemver(process.version).major < 12) {
+async function main() {
```

Comment:
All the lines added here were due to the indent with wrapping in a async function. Reason detailed below.